### DISCUSSION TITLE
Welcome to Incremental-Adventure Discussions!

### DISCUSSION THREAD

**Discussion by Flyspeck101** [Flyspeck101/Incremental-Adventure-Genesis#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to ask questions about this project. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Share your thoughts.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you want in this project.

**Comment by alahacker123** [comment]
hi nice great work

**Reply by Flyspeck101** [reply]
up me
